join/v0.2/join-v0.2.graphql

```diff
@@ -0,0 +1,25 @@
+enum Graph
+
+directive @type(
+  graph: Graph!,
+  key: String!,
```

Comment:
Technically the type of `key` here and of `requires` and `provides` in `@field` is a `FieldSet` scalar that is part of the spec too.

But also, `key` is nullable in join v0.2.